addons/web/static/src/core/domain_selector/domain_selector_branch_operator.xml

```diff
@@ -4,22 +4,22 @@
     <t t-name="web.DomainSelectorBranchOperator" owl="1">
         <t t-if="!props.readonly">
             <div class="btn-group o_domain_tree_operator_selector" aria-atomic="true">
-                <Dropdown togglerClass="'btn btn-sm btn-primary o_domain_tree_operator_caret'">
+                <Dropdown togglerClass="'btn-link btn-sm btn-primary py-0 px-1 o_domain_tree_operator_caret'">
```

Comment:
you're suppose to have `btn btn-link` => https://getbootstrap.com/docs/5.0/components/buttons/